action/main.py

```diff
@@ -14,9 +14,13 @@
 SRC = os.getenv("INPUT_SRC", default="")
 VERSION = os.getenv("INPUT_VERSION", default="")
 LINT = os.getenv("INPUT_LINT", default="")
-REVISION = os.getenv(
-    "INPUT_REVISION", default=os.getenv("INPUT_COMMIT_RANGE", default="HEAD^")
-)
+
+REVISION = os.getenv("INPUT_REVISION")
+COMMIT_RANGE = os.getenv("INPUT_COMMIT_RANGE")
+if not REVISION:
+    if not COMMIT_RANGE:
+        COMMIT_RANGE = "HEAD^"
+    REVISION = COMMIT_RANGE
```

Comment:
`COMMIT_RANGE` isn't used anywhere, so I guess this could be rewritten as e.g.
```suggestion
    REVISION = os.getenv("INPUT_COMMIT_RANGE") or "HEAD^"
```
and also line 19 could be deleted?

I also now thought about all-whitespace values (which wouldn't be False values), but it seems the YAML code `key:<space><space><space>` is decoded as {"key": None}, so that really isn't a concern.